openassessment/xblock/ui_mixins/mfe/constants.py

```diff
@@ -13,4 +13,12 @@ class ErrorCodes:
     SUBMISSION_TOO_LONG = "ERR_SUBMISSION_TOO_LONG"
     SUBMISSION_API_ERROR = "ERR_SUBMISSION_API"
     EMPTY_ANSWER = "ERR_EMPTY_ANSWER"
+    DELETE_NOT_ALLOWED = "ERR_DELETE_NOT_ALLOWED"
     UNKNOWN_ERROR = "ERR_UNKNOWN"
+
+
+class HandlerSuffixes:
+    SUBMISSION_DRAFT = 'draft'
+    SUBMISSION_SUBMIT = 'submit'
+    FILE_UPLOAD = 'upload'
+    FILE_DELETE = 'delete'
```

Comment:
Any reason to make this a `class` instead of an `Enum`? Because this feels more like an enum.